spago.yaml

```
@@ -120,4 +150,148 @@ workspace:
         - prelude
       git: https://github.com/klntsky/purescript-search-trie.git
       ref: e7f7f22486a1dba22171ec885dbc2149dc815119
-
+    language-cst-parser:
+      git: https://github.com/natefaubion/purescript-language-cst-parser.git
+      ref: "v0.13.0"
+      dependencies:
+        - "arrays"
+        - "console"
+        - "const"
+        - "control"
+        - "effect"
+        - "either"
+        - "enums"
+        - "foldable-traversable"
+        - "free"
+        - "functions"
+        - "functors"
+        - "identity"
+        - "integers"
+        - "lazy"
+        - "lists"
+        - "maybe"
+        - "newtype"
+        - "node-process"
+        - "numbers"
+        - "ordered-collections"
+        - "partial"
+        - "prelude"
+        - "st"
+        - "strings"
+        - "transformers"
+        - "tuples"
+        - "typelevel-prelude"
+        - "unfoldable"
+        - "unsafe-coerce"
+    tidy-codegen:
+      git: https://github.com/natefaubion/purescript-tidy-codegen.git
+      ref: v4.0.0
+      dependencies:
+        - aff
+        - ansi
+        - arrays
+        - avar
+        - bifunctors
+        - console
+        - control
+        - dodo-printer
+        - effect
+        - either
+        - enums
+        - exceptions
+        - filterable
+        - foldable-traversable
+        - free
+        - identity
+        - integers
+        - language-cst-parser
+        - lazy
+        - lists
+        - maybe
+        - newtype
+        - node-buffer
+        - node-child-process
+        - node-fs-aff
+        - node-path
+        - node-process
+        - node-streams
+        - ordered-collections
+        - parallel
+        - partial
+        - posix-types
+        - prelude
+        - record
+        - safe-coerce
+        - st
+        - strings
+        - tidy
+        - transformers
+        - tuples
+        - type-equality
+        - unicode
+    tidy:
+      git: https://github.com/natefaubion/purescript-tidy.git
+      ref: "v0.10.0"
+      dependencies:
+        - arrays
+        - control
+        - dodo-printer
+        - either
+        - foldable-traversable
+        - lists
+        - maybe
+        - newtype
+        - ordered-collections
+        - partial
+        - prelude
+        - language-cst-parser
+        - strings
+        - tuples
+    json-codecs:
+      git: https://github.com/JordanMartinez/purescript-json-codecs.git
+      ref: v4.0.0
+      dependencies:
+        - argonaut-core
+        - arrays
+        - bifunctors
+        - either
+        - foldable-traversable
+        - foreign-object
+        - functions
+        - identity
+        - integers
+        - lists
+        - maybe
+        - newtype
+        - nonempty
+        - nullable
+        - ordered-collections
+        - partial
+        - prelude
+        - record
+        - safe-coerce
+        - strings
+        - these
+        - tuples
+        - unsafe-coerce
```

This one is in the registry. It can just be:

```suggestion
    json-codecs: "4.0.0"
```

You can use `spago registry search $PACKAGENAME` to see if packages are in the registry